ubuntu/main.cfg

```diff
@@ -43,8 +43,16 @@ ubiquity ubiquity/success_command string \
   in-target apt --fix-broken install;\
   in-target apt upgrade;\
   in-target apt remove -y gnome-initial-setup;\
-  in-target apt install -y open-vm-tools
-  in-target apt install -y openssh-server
+  in-target apt install -y open-vm-tools;\
+  in-target apt install -y openssh-server;\
+  in-target systemctl start ssh && systemctl enable ssh;\
+  in-target sed -i 's/^[[:space:]#]*PasswordAuthentication.*/PasswordAuthentication yes/' /etc/ssh/sshd_config;\
+  in-target systemctl reload ssh;\
+  in-target apt install -y dbus-x11;\
+  in-target gsettings set org.gnome.SessionManager logout-prompt false;\
+  in-target sed -i 's/\(^GRUB_CMDLINE_LINUX_DEFAULT=".*\)"/\1 console=tty0 console=ttyS0,115200"/' /etc/default/grub
+  in-target echo "GRUB_TIMEOUT=5" >> /etc/default/grub
```

Comment:
@Psotas, please, resolve this thread if you are satisfied with the changes. Lets continue discussion in comment https://github.com/Dasharo/preseeds/pull/11#discussion_r1412096143 if it will be needed.

Reply:
Fixed here 3717eb4603ade889eaecbd61b3106cb6163519d8. Verification results, serial output from freshly installed Ubuntu:
```bash
(...)
[  OK  ] Started Disk Manager.
udisks2.service
networkd-dispatcher.service
systemd-hostnamed.service
NetworkManager-dispatcher.service
run-user-128.mount
user-runtime-dir@128.service
NetworkManager-wait-online.service
cups-browsed.service
whoopsie.service
kerneloops.service
user@128.service
rtkit-daemon.service
run-user-128-gvfs.mount
tmp-syscheck\x2dmountpoint\x2d666915338.mount
snapd.service
upower.service
systemd-localed.service
packagekit.service
geoclue.service
fprintd.service

Ubuntu 22.04.4 LTS 3mdeb ttyS0

3mdeb login:
```